SPECS/xapi.spec

```diff
@@ -472,6 +473,8 @@ Coverage files from unit tests
 - (Create plugged vifs and vbds and suspended vms)
 - Rediff xapi-1.249.3-open-openflow-port.XCP-ng.patch
 - (Open OpenFlow port for SDN controller)
+- Add xapi-1.249.3-update-db-tunnel-protocol-from-other_config.XCP-ng.patch
```

Comment:
Now that the package has had a built, a new changelog entry is necessary, with an incremented release number